### PR TITLE
convert a list to np.array to avoid warning messages

### PR DESCRIPTION
**Pull request by dtnguyen2**
# Note
This PR fixes issue #723 (a lot of warnings while running 'resample_data()' in CIAO4.12 Sherpa)

### CODE REVIEW

**Comment by DougBurke** [comment]
The macOS test had failed, but this was due to a (hopefully) random error - namely

```checking whether build environment is sane... configure: error: newly created file is older than distributed files!```

so I've re-started it (it was the only one marked as having failed).

File: sherpa/sim/tests/test_asymmetric.py [hunk 1]
```diff
@@ -182,3 +183,15 @@ def test_ui(self, tol=1.0e-3):
                                   sample['p1.gamma'])
         self.assertEqualWithinTol(self._resample_bench_10['p1.ampl'],
                                   sample['p1.ampl'])
+
+    def test_warning(self):
```
Comment (DougBurke) [inline sherpa/sim/tests/test_asymmetric.py]: @dtnguyen2 - is there any reason you didn't do the fit using the ui layer? Not that it makes a difference here,but it looks a bit odd. That is

    ui.load_ascii_with_errors(1, self.gro_fname)
    ui.set_model(ui.powlaw1d.pl)
    ui.fit()
    with warnings.bla as w:
        more-blah

Comment (dtnguyen2) [reply sherpa/sim/tests/test_asymmetric.py]: no real reason, other then laziness by cut and paste the code from the original bug report.  Want me to use the ui?

Comment (DougBurke) [reply sherpa/sim/tests/test_asymmetric.py]: No - I don't think so. It just looks a bit odd but I don't think it's a major issue (unless there was something that needed it done this way, but there isn't. So it isn't).

**Pull request merged**